src/main/java/com/exasol/releasedroid/usecases/release/ReleaseInteractor.java

```diff
@@ -52,13 +56,17 @@ public ReleaseInteractor(final ValidateUseCase validateUseCase,
     private Report makeRelease(final String repositoryFullName, final Set<PlatformName> platformNames) {
         final Report report = ReportImpl.releaseReport();
         final Repository repository = this.repositoryGateway.getRepositoryWithDefaultBranch(repositoryFullName);
-        for (final PlatformName platformName : platformNames) {
-            try {
-                this.getReleaseMaker(platformName).makeRelease(repository);
-                report.addResult(ReleaseResult.successfulRelease(platformName));
-            } catch (final ReleaseException exception) {
-                report.addResult(ReleaseResult.failedRelease(platformName, ExceptionUtils.getStackTrace(exception)));
-                break;
+        for (final PlatformName platformName : this.releasePriority) {
```

Comment:
it's a little bit tricky to understand the whole flow. I suggest to extract this to a dedicated method, something like `List<PlatformName> orderByPriority(platformNames)` or similar